.ruff.toml

```diff
@@ -22,6 +22,7 @@ select = [
 ]
 ignore = [
   "E402",  # module import not at top of file
+  "E501", # line too long
```

Comment:
We shouldn’t be ignoring this globally, we can maybe remove this for a specific line but I think we should revert this change @hoodmane

Reply:
There's a separate `ruff fmt` that formats code. If the formatter makes a long line I think it's very frustrating for `ruff`'s linter to then complain about it.